.github/workflows/bazelbuild.yml

```diff
@@ -0,0 +1,16 @@
+name: Bazel Build
+
+on: [pull_request, push]
+
+jobs:
+  macos:
+    name: Bazel
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Test iOS
+      run: bazelisk test //Tests:KronosTestsiOS --apple_platform_type=ios
```

Comment:
I wonder if we should lock the bazel version with a `.bazelversion` file. It would make this all more reproducible but would give us another thing we have to update occasionally